test/codec-utils.test.js

```diff
@@ -0,0 +1,28 @@
+import Qunit from 'qunit';
+import { translateLegacyCodecs } from '../src/codec-utils';
+
+const { deepEqual, module, test } = Qunit;
```

Comment:
I actually just copied it from https://github.com/videojs/videojs-contrib-media-sources/blob/master/test/add-text-track-data.test.js , but I can change it back.